raiden/transfer/views.py

```diff
@@ -58,6 +61,28 @@ def block_number(chain_state: ChainState) -> BlockNumber:
     return chain_state.block_number
 
 
+def latest_confirmed_block_hash(
+        chain_state: ChainState,
+        web3: Web3,
```

Comment:
Please don't add a function in the views that does IO. This can go into a proxy or just be a direct call to web3.

Reply:
Agreed.